src/modules/colorPicker/ColorPickerUI/Controls/ColorPickerControl.xaml

```diff
@@ -370,18 +369,20 @@
                             TextChanged="HexCode_TextChanged"
                             TextWrapping="Wrap" />
 
-                        <ui:Button
+                            <Button
                             x:Name="OKButton"
                             Grid.Row="9"
                             Grid.ColumnSpan="4"
                             Margin="0,32,0,0"
                             HorizontalAlignment="Stretch"
-                            Appearance="Primary"
+                            Background="{DynamicResource {x:Static SystemColors.AccentColorBrushKey}}"
```

Comment:
Is it possible to use `{StaticResource}` with a value from WinUI Gallery?

Reply:
We don't use WinUI but .net WPF.